### PR TITLE
feat(icons): Add toilet icon

### LINKED ISSUE #1788: Toilet
### Icon name

toilet

### Use cases

Showing that a rental room or cottage has access to its own toilet

### Design ideas

toilet

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)
- [X] I have searched existing icons to make sure it does not already exist and I didn't find any.
- [X] I am not requesting a brand logo and the art is not protected by copyright.
- [X] I am not requesting an icon that includes religious, political imagery or hate symbols.
- [X] I have provided appropriate use cases for the icon(s) requested.

### PR DESCRIPTION
**Pull request by EthanHazel**
Closes #1788

## What is the purpose of this pull request?

- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Description

Creates a toilet icon SVG and JSON.

### Icon use case 

Can be used for things such as house / hotel listings to show the amount of bathrooms available.
Health related apps that could use it for medical related reasonings.
Maps showing you where the nearest bathroom is located.

### Concept 

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [x] The icons are solely my own creation.
- [ ] The icons were originally created in #<issueNumber> by @<githubUser>
- [ ] I've based them on the following Lucide icons: 
- [ ] I've based them on the following design: 

### Naming 

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to three points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

File: icons/toilet.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+    <path d="M7,12h13s1,0,1,1c0,4-4,5-7,5,0,0,2,2,2,3s-1,1-1,1H6s-1,0-1-1,2-3,2-3c0,0-4-1-4-5V4c0-1,1-2,2-2H13c1,0,2,1,2,2V12"/>
```
Comment (ericfennis) [inline icons/toilet.svg:12]: Can you format this code?
Comas should be replaced with spaces

**Review by ericfennis — CHANGES_REQUESTED** [review]

**Comment by gorgobacka** [comment]
@EthanHazel Are you going to fix this?

@ericfennis Is the linting the only point that needs to be adjusted? And would it be possible to re-submit this PR using the same icon, if nothing happens in a reasonable time?  I would like to see this icon in another app that is using this package.

**Comment by EthanHazel** [comment]
@ericfennis Just fixed the formatting of the SVG, sorry for the late reply

**Comment by jguddas** [comment]
More arcs:
<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M7+12h13a1+1+0+0+1+1+1+5+5+0+0+1-5+5h-.598a.5.5+0+0+0-.424.765l1.544+2.47a.5.5+0+0+1-.424.765H5.402a.5.5+0+0+1-.424-.765L7+18%22+/%3E%3Cpath+d=%22M8+18a5+5+0+0+1-5-5V4a2+2+0+0+1+2-2h8a2+2+0+0+1+2+2v8%22+/%3E">icons<br/>Open lucide studio</a>

**Comment by EthanHazel** [comment]
> More arcs: [iconsOpen lucide studio](https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M7+12h13a1+1+0+0+1+1+1+5+5+0+0+1-5+5h-.598a.5.5+0+0+0-.424.765l1.544+2.47a.5.5+0+0+1-.424.765H5.402a.5.5+0+0+1-.424-.765L7+18%22+/%3E%3Cpath+d=%22M8+18a5+5+0+0+1-5-5V4a2+2+0+0+1+2-2h8a2+2+0+0+1+2+2v8%22+/%3E)

Looks great! Added this version to the PR and added you to contributors

**Review by EthanHazel — COMMENTED** [review]
Fixed issues related to naming and tags

**Comment by ericfennis** [comment]
@EthanHazel Can you check the linter?

**Comment by EthanHazel** [comment]
> @EthanHazel Can you check the linter?

Just ran the prettier script and should pass fine now

**Review by ericfennis — APPROVED** [review]
Nice looks good now

**Review by EthanHazel — COMMENTED** [review]
test

**Review by jguddas — APPROVED** [review]

**Pull request merged**